e2e-tests/test_runner.sh

```diff
@@ -49,13 +51,20 @@ if [ ! -f bin/zombienet-linux-x64 ]; then
 fi
 
 # Check if we requested a run over a debug build
-if [[ "$@" == *"--debug"* ]]
+BUILDSUBPATH="release"
+for ARG in "$@"; do
+    if [[ "${ARG}" == "--debug" ]]
+    then
+        BUILDSUBPATH="debug"
+    fi
+done
+
+if [[ ${BUILDSUBPATH} == "release" ]]; 
 then
-    echo -e "${TXT_BIGRN}INFO: ${TXT_BIBLK}Running tests with a debug build${TXT_NORML}"
-    BUILDSUBPATH="debug"
-else
     echo -e "${TXT_BIGRN}INFO: ${TXT_BIBLK}Running tests with a release build${TXT_NORML}"
-    BUILDSUBPATH="release"
+elif [[ ${BUILDSUBPATH} == "debug" ]];
+then
+    echo -e "${TXT_BIGRN}INFO: ${TXT_BIBLK}Running tests with a debug build${TXT_NORML}"
 fi
```

Comment:
It looks like we could replace lines 62-68 with a single one:

```suggestion
echo -e "${TXT_BIGRN}INFO: ${TXT_BIBLK}Running tests with a ${BUILDSUBPATH} build${TXT_NORML}"
```